Dockerfile

```diff
@@ -21,6 +21,7 @@ WORKDIR /app
 
 COPY --from=deps /app/npm ./npm
 COPY --from=deps /app/node_modules ./node_modules
+COPY --from=deps /app/migrate.sh ./migrate.sh
```

Comment:
This can be removed as the copy on the next line takes care of moving all the files from the build context.